inference-engine/tests_deprecated/helpers/mock_icore.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <gmock/gmock.h>
+#include "ie_icore.hpp"
+
+class MockICore : public InferenceEngine::ICore {
```

Comment:
@iefode do we need to double `MockICore` inside the deprecated tests? I supposed we can re-use from `inference-engine/tests/ie_test_utils/unit_test_utils/mocks/cpp_interfaces/interface/mock_icore.hpp`

Reply:
I agree with your point, we can reuse it